tests/test_bgbb_udfs.py

```diff
@@ -0,0 +1,78 @@
+from lifetimes.datasets import load_donations
+import pyspark
+from pytest import fixture
+
+from bgbb import BGBB
+from bgbb.sql.bgbb_udfs import mk_udfs
+
+spark = pyspark.sql.SparkSession.builder.appName("test").getOrCreate()
```

Comment:
Might be useful to have this as a fixture so the context can be stopped at the end, or shared across multiple tests. This is probably fine since all the spark tests are isolated to this file.

```suggestion
@fixture
def spark():
    spark = pyspark.sql.SparkSession.builder.appName("test").getOrCreate()
    yield spark
    spark.stop()
```